### PR TITLE
Add extra monitoring indicators part1

### PR DESCRIPTION
**Pull request by yyongycy**
### What changes are proposed in this pull request?

Add some monitoring indicators for the master status

### Why are the changes needed?
Add some monitoring indicators for the master status, collecting information for further system throttling.

### Does this PR introduce any user facing changes?
NA

### CODE REVIEW

**Comment by yyongycy** [comment]
Following ones: https://github.com/Alluxio/alluxio/pull/16173 @beinan

**Review by beinan — APPROVED** [review]
as we discussed offline, there is no behavior change except to add more monitoring for this pr, so it looks good to me

File: core/common/src/main/java/alluxio/exception/BusyException.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+package alluxio.exception;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Thrown when the system is busy.
+ */
+@ThreadSafe
+public class BusyException extends AlluxioException {
```
[the remark this answers is not in the document]
Comment (yyongycy) [reply core/common/src/main/java/alluxio/exception/BusyException.java:20]: This is an exception. No actual handling code.

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Pull request merged**